pkg/ingester/shipper.go

```diff
@@ -197,7 +197,8 @@ func (s *Shipper) Sync(ctx context.Context) (shipped int, err error) {
 // This updated version of meta.json is however not persisted locally on the disk, to avoid race condition when TSDB
 // library could actually unload the block if it found meta.json file missing.
 func (s *Shipper) upload(ctx context.Context, meta *metadata.Meta) error {
-	level.Info(s.logger).Log("msg", "upload new block", "id", meta.ULID)
+	level.Info(s.logger).Log("msg", "uploading new block", "id", meta.ULID)
+	defer level.Info(s.logger).Log("msg", "finished uploading new block", "id", meta.ULID)
```

Comment:
I would not defer this, because it could have failed, or even worse, panicked.
Given the caller of this on the line 169 already logs when shipping failed, maybe we could move this log there, when it didn't fail?

Reply:
sounds reasonable. I can also log it there 👍